src/server/game/Miscellaneous/SharedDefines.h

```diff
@@ -6508,4 +6508,19 @@ enum class GameError : uint32
     ERR_NOT_IN_PET_BATTLE                                           = 1047,
 };
 
+enum class MountResult : uint32
+{
+    InvalidMountee = 0,
+    TooFarAway     = 1,
+    AlreadyMounted = 2,
+    NotMountable   = 3,
+    NotYourPet     = 4,
+    Other          = 5,
+    Looting        = 6,
```

Comment:
How can Mount result be " Looting" ?

Reply:
These values are from wpp. These values are not only for smsg_mount_result but also for smsg_dismount_result so some of these values are used for the other opcode as well which I have not discovered in sniffs yet.

Reply:
Values are correct, comment is wrong. They are used exclusively for SMSG_MOUNT_RESULT.